src/core/pages/playground/Playground.jsx

```diff
@@ -1,51 +1,16 @@
 import React from 'react';
 
-import { Input, Button } from '../../atoms';
+import { Card } from '../../atoms';
 
 export class Playground extends React.Component {
   state = {
-    name: '',
-    nameError: false,
-    nameErrorText: [],
-    nameValid: false
+    heading: 'The Yorkshire Dales Flower Company',
+    subHeading: 'Bradley in West Yorkshire'
   };
 
-  validateForm = () => {
-    const { name } = this.state;
-
-    if (name.length < 3) {
-      this.setState({
-        nameValid: false,
-        nameError: true,
-        nameErrorText: ['Needs to be 3 characters']
-      });
-    } else {
-      this.setState({
-        nameError: false,
-        nameValid: true
-      });
-    }
-  };
-
-  updateState = state => this.setState(state);
-
   render() {
-    const { name, nameError, nameValid, nameErrorText } = this.state;
-    return (
-      <div className="u-container u-padding-top">
-        <h3 className="c-heading-tiny u-margin-bottom">3 or more characters</h3>
-        <Input
-          value={name}
-          id="name"
-          onChange={this.updateState}
-          label="Name"
-          placeholder="Enter first name"
-          error={nameError}
-          errorList={nameErrorText}
-          valid={nameValid}
-        />
-        <Button onClick={this.validateForm}>Start subscription</Button>
-      </div>
-    );
+    const { heading, subHeading } = this.state;
+
+    return <Card heading={heading} subHeading={subHeading} />;
```

Comment:
Just to see the cards a little easier with some spacing and grid action 💯 

```suggestion
    return (
      <div className="u-container u-margin-top">
        <div className="u-sm-12 u-med-6 u-lg-6 u-margin-bottom">
          <Card heading={heading} subHeading={subHeading} />
        </div>
        <div className="u-sm-12 u-med-6 u-lg-6 u-margin-bottom">
          <Card heading={heading} subHeading={subHeading} />
        </div>
      </div>
    );
```